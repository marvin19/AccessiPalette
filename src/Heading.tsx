const Heading = () => {
  return (
    <div className="container">
        <h1 className="h1"><span aria-hidden="true">🎨</span> AccessiPalette</h1>
    </div>
  )
}

export default Heading;