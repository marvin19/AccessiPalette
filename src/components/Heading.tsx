const Heading = (): JSX.Element => {
    return (
        <div className="container w-full mx-auto bg-">
            <h2 className="h1">
                <span aria-hidden="true">ðŸŽ¨</span> AccessiPalette
            </h2>
        </div>
    );
};

export default Heading;
