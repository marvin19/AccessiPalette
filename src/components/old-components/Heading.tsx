const Heading = (): JSX.Element => {
    return (
        <div className="container w-full mx-auto bg-">
            <h1 className="h1">
                <span aria-hidden="true">ðŸŽ¨</span> AccessiPalette
            </h1>
        </div>
    );
};

export default Heading;
