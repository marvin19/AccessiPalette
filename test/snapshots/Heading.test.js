import React from 'react';
import { render } from '@testing-library/react';
import Heading from '../../src/components/Heading'; // Adjust the import path as necessary

describe('Heading Component', () => {
    it('renders correctly and matches snapshot', () => {
        const { asFragment } = render(<Heading />);
        expect(asFragment()).toMatchSnapshot();
    });
});
