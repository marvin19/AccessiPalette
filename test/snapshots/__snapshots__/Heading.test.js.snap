// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading Component renders correctly and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container w-full mx-auto bg-"
  >
    <h2
      class="h1"
    >
      <span
        aria-hidden="true"
      >
        ðŸŽ¨
      </span>
       AccessiPalette
    </h2>
  </div>
</DocumentFragment>
`;
